src/jit/RegisterAlloc.cpp

```diff
@@ -799,9 +852,8 @@ void JITCompiler::allocateRegisters()
                         regs.floatReserve(reg + 1);
                     }
 #endif /* SLJIT_CONFIG_ARM_32 */
-
-                    regs.floatReserve(reg);
-                    instr->setRequiredReg(reuseTmpIndex, regs.toCPUFloatReg(reg));
+                    SHORTIF(((*nextType & Instruction::TypeMask) == Instruction::V128Operand), regs.vectorReserve(reg), regs.floatReserve(reg))
+                    SHORTIF(((*nextType & Instruction::TypeMask) == Instruction::V128Operand), instr->setRequiredReg(reuseTmpIndex, regs.toCPUVectorReg(reg)), instr->setRequiredReg(reuseTmpIndex, regs.toCPUFloatReg(reg)))
```

Comment:
You can use a `(a, b)` comma separated list for commands. This way you don't need to duplicate the macro (if).

E.g: `(regs.vectorReserve(reg), instr->setRequiredReg(reuseTmpIndex, regs.toCPUVectorReg(reg)))`